Flaky TestNumpyRandom: test_numpy_uniform_size_int, test_numpy_logistic0b

When building Pythran 0.9.7 with https://github.com/serge-sans-paille/pythran/pull/1640 applied on top, some builds failed on some architectures. Rebuilding the package again until it built "fixed" the problem.

### armv7hl

```
_________________ TestNumpyRandom.test_numpy_uniform_size_int __________________
[gw3] linux -- Python 3.9.0 /usr/bin/python3
self = <pythran.tests.test_numpy_random.TestNumpyRandom testMethod=test_numpy_uniform_size_int>
    def test_numpy_uniform_size_int(self):
        """ Check logseries with arguments with mean and variance. """
        code = """
        def numpy_uniform_size_int(size):
            import numpy as np
            from numpy.random import uniform
            low, high = 0., 1234.
            rmean = 0.5 * (low + high)
            rvar = (high - low) ** 2 / 12
            a = uniform(low, high, size)
            cond_mean = (a.mean() - rmean) / rmean < 0.05
            cond_var = (np.var(a) - rvar) / rvar < 0.05
            return cond_mean and cond_var
        """
>       self.run_test(code, 4000, numpy_uniform_size_int=[int])
pythran/tests/test_numpy_random.py:1594: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
pythran/tests/__init__.py:318: in run_test
    self.assertAlmostEqual(python_ref, pythran_res)
pythran/tests/__init__.py:144: in assertAlmostEqual
    self.assertEqual(ref, res)
E   AssertionError: True != False
----------------------------- Captured stdout call -----------------------------
Python result:  True
Pythran result:  False
Type of Pythran res :  <class 'bool'>
Type of Python ref :  <class 'numpy.bool_'>
```

And:

```
____________________ TestNumpyRandom.test_numpy_logistic0b _____________________
[gw3] linux -- Python 3.9.0 /usr/bin/python3
self = <pythran.tests.test_numpy_random.TestNumpyRandom testMethod=test_numpy_logistic0b>
    def test_numpy_logistic0b(self):
        """ Check logistic with 2 argument with mean and variance. """
        code = """
        def numpy_logistic0b(size):
            from numpy.random import logistic
            from numpy import var, mean, pi
            u = 2.
            s = 2
            rmean = u
            rvar = ((s**2*pi**2)/3)
            a = logistic(u, s, size)
            return (abs(mean(a) - rmean) < 0.05 and abs(var(a) - rvar) < .05)
        """
>       self.run_test(code, 10 ** 6, numpy_logistic0b=[int])
pythran/tests/test_numpy_random.py:1399: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
pythran/tests/__init__.py:318: in run_test
    self.assertAlmostEqual(python_ref, pythran_res)
pythran/tests/__init__.py:144: in assertAlmostEqual
    self.assertEqual(ref, res)
E   AssertionError: True != False
----------------------------- Captured stdout call -----------------------------
Python result:  True
Pythran result:  False
Type of Pythran res :  <class 'bool'>
Type of Python ref :  <class 'numpy.bool_'>
```

### aarch64

```
____________________ TestNumpyRandom.test_numpy_logistic0b _____________________
[gw1] linux -- Python 3.9.0 /usr/bin/python3
self = <pythran.tests.test_numpy_random.TestNumpyRandom testMethod=test_numpy_logistic0b>
    def test_numpy_logistic0b(self):
        """ Check logistic with 2 argument with mean and variance. """
        code = """
        def numpy_logistic0b(size):
            from numpy.random import logistic
            from numpy import var, mean, pi
            u = 2.
            s = 2
            rmean = u
            rvar = ((s**2*pi**2)/3)
            a = logistic(u, s, size)
            return (abs(mean(a) - rmean) < 0.05 and abs(var(a) - rvar) < .05)
        """
>       self.run_test(code, 10 ** 6, numpy_logistic0b=[int])
pythran/tests/test_numpy_random.py:1399: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
pythran/tests/__init__.py:318: in run_test
    self.assertAlmostEqual(python_ref, pythran_res)
pythran/tests/__init__.py:144: in assertAlmostEqual
    self.assertEqual(ref, res)
E   AssertionError: True != False
----------------------------- Captured stdout call -----------------------------
Python result:  True
Pythran result:  False
Type of Pythran res :  <class 'bool'>
Type of Python ref :  <class 'numpy.bool_'>
```

### x86_64

```
____________________ TestNumpyRandom.test_numpy_logistic0b _____________________
[gw35] linux -- Python 3.9.0 /usr/bin/python3
self = <pythran.tests.test_numpy_random.TestNumpyRandom testMethod=test_numpy_logistic0b>
    def test_numpy_logistic0b(self):
        """ Check logistic with 2 argument with mean and variance. """
        code = """
        def numpy_logistic0b(size):
            from numpy.random import logistic
            from numpy import var, mean, pi
            u = 2.
            s = 2
            rmean = u
            rvar = ((s**2*pi**2)/3)
            a = logistic(u, s, size)
            return (abs(mean(a) - rmean) < 0.05 and abs(var(a) - rvar) < .05)
        """
>       self.run_test(code, 10 ** 6, numpy_logistic0b=[int])
pythran/tests/test_numpy_random.py:1399: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
pythran/tests/__init__.py:318: in run_test
    self.assertAlmostEqual(python_ref, pythran_res)
pythran/tests/__init__.py:144: in assertAlmostEqual
    self.assertEqual(ref, res)
E   AssertionError: True != False
----------------------------- Captured stdout call -----------------------------
Python result:  True
Pythran result:  False
Type of Pythran res :  <class 'bool'>
Type of Python ref :  <class 'numpy.bool_'>
```

--- a/pythran/tests/test_numpy_random.py
+++ b/pythran/tests/test_numpy_random.py
@@ -35,7 +35,7 @@ class TestNumpyRandom(TestEnv):
                 from numpy.random import random
                 from numpy import sum
                 s = sum(random((n, n)))
-                return (abs(s / (n * n) - .5) < 5e-3)""",
+                return (abs(s / (n * n) - .5) < .05)""",
                       10 ** 3, numpy_random2=[int])
 
     ###########################################################################
@@ -67,7 +67,7 @@ class TestNumpyRandom(TestEnv):
                 from numpy.random import random_sample
                 from numpy import sum
                 s = sum(random_sample((n, n)))
-                return (abs(s / (n * n) - .5) < 5e-3)""",
+                return (abs(s / (n * n) - .5) < .05)""",
                       10 ** 3, numpy_random_sample2=[int])
 
     ###########################################################################
@@ -101,7 +101,7 @@ class TestNumpyRandom(TestEnv):
                 from numpy.random import ranf
                 from numpy import mean
                 a = ranf((n, n))
-                return (abs(mean(a) - .5) < 5e-3)""",
+                return (abs(mean(a) - .5) < .05)""",
                       10 ** 3, numpy_ranf2=[int])
 
     ###########################################################################
@@ -135,7 +135,7 @@ class TestNumpyRandom(TestEnv):
                 from numpy.random import sample
                 from numpy import mean
                 s = sample((n, n))
-                return (abs(mean(s) - .5) < 5e-3)""",
+                return (abs(mean(s) - .5) < .05)""",
                       10 ** 3, numpy_sample2=[int])
 
     ###########################################################################
@@ -158,7 +158,7 @@ class TestNumpyRandom(TestEnv):
                 from numpy.random import rand
                 from numpy import sum
                 s = sum(rand(n, n))
-                return (abs(s / (n * n) - .5) < 5e-3)""",
+                return (abs(s / (n * n) - .5) < .05)""",
                       10 ** 3, numpy_rand1=[int])
 
     ###########################################################################
@@ -287,7 +287,7 @@ class TestNumpyRandom(TestEnv):
             mu, sigma = 0, 0.1
             a = normal(mu, sigma, size)
             print(mean(a))
-            return (abs(mu - mean(a)) < 0.01 and abs(sigma - sqrt(var(a,ddof=1))) < .01)
+            return (abs(mu - mean(a)) < 0.05 and abs(sigma - sqrt(var(a,ddof=1))) < .05)
         """
         self.run_test(code, 10 ** 5, numpy_normal0b=[int])
 
@@ -830,7 +830,7 @@ class TestNumpyRandom(TestEnv):
             s = 1/2
             rmean = e**(m+(s**2/2))  
             rvar = (e**(s**2) - 1)*e**(2*m+s**2) 
-            return (abs(mean(a) - rmean) < .2 and abs(var(a) - rvar) < .2)
+            return (abs(mean(a) - rmean) < .1 and abs(var(a) - rvar) < .1)
         """
         self.run_test(code, 10 ** 6, numpy_lognormal0=[int])
 
@@ -845,7 +845,7 @@ class TestNumpyRandom(TestEnv):
             a = [lognormal(m) for x in range(size)]
             rmean = e**(m+(s**2/2))  
             rvar = (e**(s**2) - 1)*e**(2*m+s**2) 
-            return (abs(mean(a)- rmean) < 0.2 and abs(var(a) - rvar) < .2)
+            return (abs(mean(a)- rmean) < 0.1 and abs(var(a) - rvar) < .1)
         """
         self.run_test(code, 10 ** 6, numpy_lognormal0a=[int])
 
@@ -875,7 +875,7 @@ class TestNumpyRandom(TestEnv):
             rmean = e**(m+(s**2/2))  
             rvar = (e**(s**2) - 1)*e**(2*m+s**2) 
             a = lognormal(size=size)
-            return (abs(mean(a) - rmean) < .1 and abs(var(a) - rvar) < .2)
+            return (abs(mean(a) - rmean) < .1 and abs(var(a) - rvar) < .1)
         """
         self.run_test(code, 10 ** 6, numpy_lognormal1=[int])
 

